app/main/views/briefs.py

```diff
@@ -81,6 +84,10 @@ def ask_brief_clarification_question(brief_id):
 def start_brief_response(brief_id):
     brief = get_brief(data_api_client, brief_id, allowed_statuses=['live'])
 
+    if not (datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d")
```

Comment:
It looks like there is a test for it that passes but if I ran this just in my terminal I think it would break?
![image](https://cloud.githubusercontent.com/assets/7228605/21151834/dfa2c09e-c15b-11e6-8ae1-967d73afd6c0.png)